app/models/repository.js

```diff
@@ -3,11 +3,17 @@ import DS from 'ember-data';
 
 export default DS.Model.extend({
   name: DS.attr('string'),
-  slug: DS.attr('string'),
   description: DS.attr('string'),
   namespace: DS.belongsTo(),
+  "content_type": DS.attr('string'),
+  "public_access": DS.attr('boolean'),
+
   repoSlug: Ember.computed('id', function() {
     let f = this.get('id').split('/');
     return f[1];
-  })
+  }),
+
+  splitRepoSlug: function() {
```

Comment:
Do we need the `Repo` in the name? Since we are in the repository model, why don't we just call it `splitSlug`? Or even better `splitID` because in terms of the frontend, the slug *is* the ID. (slug is just a backend-internal thing which acts as an identifier to the outside.)